phpstan.neon

```diff
@@ -1,7 +1,5 @@
 includes:
     - vendor/jangregor/phpstan-prophecy/extension.neon
-    - vendor/phpstan/phpstan-doctrine/extension.neon
```

Comment:
this bundle does not use doctrine

Reply:
decided to add this again because it should not hurt 🙈

Reply:
can not see that its added ?